ledger-api/grpc-definitions/com/daml/ledger/api/v1/active_contracts_service.proto

```diff
@@ -15,6 +15,11 @@ option java_package = "com.daml.ledger.api.v1";
 option csharp_namespace = "Com.Daml.Ledger.Api.V1";
 
 // Allows clients to initialize themselves according to a fairly recent state of the ledger without reading through all transactions that were committed since the ledger's creation.
+//
+// When the participant is run in mode requiring authentication, all the calls
+// in this interface will respond with ``UNAUTHENTICATED``, if the caller fails
+// to provide a valid access token, and will respond with ``PERMISSION_DENIED``, if
+// the claims in the token are insufficient to perform a given operation.
```

Comment:
```suggestion
// When the participant is run with authentication enabled, any call to this service
// will return ``UNAUTHENTICATED`` if the caller doesn't provide a valid access token
// and ``PERMISSION_DENIED`` if the claims in the token are insufficient.
```

Reply:
Also, what do back-ticks do?

Reply:
It's supposed to be rendered using a fixed width font.

Reply:
Here: https://docs.daml.com/1.6.0/app-dev/grpc/proto-docs.html